pkg/webhookconfig/registration.go

```diff
@@ -396,9 +400,10 @@ func (wrc *Register) checkEndpoint() error {
 	deploy, err := wrc.GetKubePolicyDeployment()
 	if err != nil {
 		return err
-	}
-	if tlsutils.IsKyvernoInRollingUpdate(deploy) {
-		return errors.New("kyverno is in rolling update, please update the timeout by setting the webhookRegistrationTimeout flag")
+	} else {
```

Comment:
No need to wrap it in an else as we return on the err above.

Reply:
fixed